tests/integration/vrf/04-multi-vrf-loopback-ipv6.yml

```diff
@@ -21,14 +21,16 @@ groups:
 
 vrfs:
   red:
-    loopback:
-      ipv6: 2001:db8:c001:cafe::/64
     links: [ dut-h1, dut-h2 ]
   blue:
-    loopback:
-      ipv6: 2001:db8:c001:cafe::/64
     links: [ dut-h3, dut-h4 ]
 
+nodes:
+  dut:
+    vrfs:
+      red.loopback.ipv6: 2001:db8:c001:cafe::/64
+      blue.loopback.ipv6: 2001:db8:c001:cafe::/64
```

Comment:
It could be useful to allow ```vrf.loopback``` to be a pair of prefixes in addition to a bool, such that the user could do:

```
nodes:
  dut:
    vrf.loopback:
      ipv4: False
      ipv6: 2001:db8:c001:cafe::/64
```

Having the same loopback IP replicated in every VRF seems common?

Reply:
> It could be useful to allow `vrf.loopback` to be a pair of prefixes in addition to a bool, such that the user could do:

A) You can do that if you wish (or my .ipv6 stuff wouldn't work)
B) There's no need to set .ipv4 to False. If you specify a prefix for a node loopback, it's not merged with anything else.

> Having the same loopback IP replicated in every VRF seems common?

I wouldn't do that, but... 🤷‍♂️